config_templates/gretel/synthetics/default.yml

```diff
@@ -1,7 +1,9 @@
 # Default configuration for Synthetic model creation.
 
 # The parameter settings below match the default settings 
-# in Gretel's open source synthetic package
+# in Gretel's open source synthetics package
```

Comment:
Nit: suggest changing this to SAL language

Reply:
done